modules/xfem/include/base/XFEM.h

```diff
@@ -66,7 +66,7 @@ class XFEM : public XFEMInterface
 
   ~XFEM();
 
-  void addGeometricCut(const GeometricCutUserObject * geometric_cut);
+  void addGeometricCut(GeometricCutUserObject * geometric_cut);
```

Comment:
It looks like you missed adding this one.

Reply:
I removed it because I need to set an interface id in this function to geometric_cut and the set function is not a const function. Do you have a better way to do it?

Reply:
I just meant that you didn't add the Doxygen comment. Since this method already existed, I'm not going to let that hold up the merging of this.